Update Slate project status - Low maintenance (#1001)
Shopify/slate: README.md

![slate animated banner](https://user-images.githubusercontent.com/4837696/47506317-cbe22400-d83d-11e8-9867-1dc874943833.gif)

## Slate on Low Maintenance

> Due to project priority and resource constraints, this project is currently on low maintenance. We recognize that there are a lot of activities around this package, however, we unable to accommodate the maintenance this project requires.

Reviewer: ```suggestion
Due to project priority and resource constraints, this project is currently on low maintenance. We recognize that there are a lot of activities around this package. However, we are unable to accommodate the maintenance this project requires.
```

### What does this mean?
Slate will not have new features or bug fixes during this period.

As a Shopify repository maintainer, there will be no activities around:

- New feature implementation
- Existing feature changes
- Major bugs due to any tools version changes
- Major bugs due to Shopify internal changes
- Minor bugs
- Will not evaluate and merge any community created PRs
- Will not address any community reported issues except ones that implicate Shopify security

### Should I continue using Slate?

> You may. As it is, Slate is still useful for themes development.

Reviewer: ```suggestion
You may. As it is, Slate is still useful for Shopify theme development.
```

### What happened? Why are you stopping? Will Shopify work on Slate in the future?

> Due to Shopify priorities and resource constraints, we have decided to put more emphasis on another project rather than Slate. We have plans to return back to Slate when our priorities shifts back to Slate. While evaluating the current state of Slate, there are two key developer workflows that we are not satisfied with:

Reviewer: ```suggestion
Due to Shopify priorities and resource constraints, we have decided to put more emphasis on another projects. We have plans to return to Slate once our priorities change. While evaluating the current state of Slate, there are two key developer workflows that we are not satisfied with:
```
